test/parallel/test-buffer-fill.js

```diff
@@ -325,11 +324,6 @@ Buffer.alloc(8, '');
   assert.strictEqual(buf.toString(), 'էէէէէ');
 }
 
-// Testing process.binding. Make sure "start" is properly checked for -1 wrap
-// around.
-assert.strictEqual(
-  internalBinding('buffer').fill(Buffer.alloc(1), 1, -1, 0, 1), -2);
-
```

Comment:
Rather than removing the `assert.strictEqual()` here, these should check that they throw with an appropriate error code. Check the tests for other examples of using `assert.throws()`.

Reply:
I converted this and the other instance to minimal tests. PTAL.